services/graphql-server/src/graphql/resolvers/platform/content.js

```diff
@@ -918,8 +918,18 @@ module.exports = {
         excludeContentTypes,
       });
 
-      customAttributes.forEach(({ key, value }) => {
-        query.$and.push({ [`customAttributes.${key}`]: value });
+      customAttributes.forEach(({ key, value, exists }) => {
+        const valueOrExists = Boolean(value || typeof exists !== 'undefined');
+        if (!valueOrExists) {
+          throw new UserInputError('Either value or exists must be defined');
+        } else {
```

Comment:
Use condensed if here if it will fit; `if(!validInput) throw new UserInputError(...);`. No need to wrap the rest in an else since the throw will abort execution of the rest.